base/applications/mspaint/registry.cpp

```diff
@@ -61,6 +61,23 @@ void RegistrySettings::LoadPresets()
     ThumbXPos = 180;
     ThumbYPos = 200;
     UnitSetting = 0;
+    Bold = FALSE;
+    Italic = FALSE;
+    Underline = FALSE;
+    CharSet = DEFAULT_CHARSET;
+    PointSize = 14;
+    FontsPositionX = 0;
+    FontsPositionY = 0;
+    ShowTextTool = TRUE;
+
+    {
+        LOGFONT lf;
+        GetObject(GetStockObject(DEFAULT_GUI_FONT), sizeof(lf), &lf);
+        strFontName = lf.lfFaceName;
+    }
+
+    WINDOWPLACEMENT WindowPlacement;
```

Comment:
What is this for? The variable doesn't seem to be used anywhere else; the assignation value below doesn't seem to do anything useful afterwards.

Reply:
In `main.cpp`:
```cxx
    /* by moving the window, the things in WM_SIZE are done */
    mainWindow.SetWindowPlacement(&(registrySettings.WindowPlacement));
```

Reply:
But here in this line 79 in `RegistrySettings::LoadPresets`, this is not a member, this is a local variable to this function.

Reply:
Fixed.